bucket/djvulibre.json

```
@@ -0,0 +1,42 @@
+{
+    "version": "3.5.27-4.11",
+    "description": "DjVu library.",
```

This is not much optimal description.

```suggestion
    "description": "Web-centric format and software platform for distributing documents and images.",
```